EIPS/eip-1193.md

```diff
@@ -27,16 +27,19 @@ The events `connect`, `disconnect`, `chainChanged`, and `accountsChanged` are pr
 Makes an Ethereum RPC method call.
 
 ```typescript
-type RequestParams = Array<any> | { [key: string]: any };
+interface RequestArguments {
+  method: string;
+  params?: Array<any> | { [key: string]: any };
```

Comment:
`any` should be `unknown`.  Never use `any` in TypeScript, it basically is a shorthand for "turn type checking off entirely for this variable".

Reply:
Done in 7753ce8